### PR TITLE
Fix eos_acls documentation errors

### PR DESCRIPTION
**Pull request by dmahler**
##### SUMMARY
Fix errors in Arista ACLs readme found after testing.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
arista.eos.eos_acls

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: docs/arista.eos.eos_acls_module.rst [hunk 1]
```diff
@@ -3016,35 +3016,6 @@ Examples
...
-    #    10 permit ip 10.10.10.0/24 any ttl eq 200
-    #    20 permit ip 10.30.10.0/24 host 10.20.10.1
-    #    30 deny tcp host 10.10.20.1 eq finger www any syn log
-    #    40 permit ip any any
-    # ipv6 access-list test2
-    #     10 deny icmpv6 any any reject-route hop-limit eq 20
-
-    # !
-
-    - name: Delete provided configuration
```
Comment (dmahler) [inline docs/arista.eos.eos_acls_module.rst:3032]: This is formatted incorrectly (the ACL to delete is not named), the correct one is below - looks like it was fixed in the next section but this incorrect one was left in.

File: docs/arista.eos.eos_acls_module.rst [hunk 2]
```diff
@@ -2998,7 +2998,7 @@ Examples
           - name: test1
             aces:
             - sequence: 35
-              action: permit
```
Comment (dmahler) [inline docs/arista.eos.eos_acls_module.rst:3001]: action is not a valid param, it's `grant`

**Comment by dmahler** [comment]
cc @GomathiselviS

**Comment by GomathiselviS** [comment]
@dmahler Thanks for working on this.  Can you please add a changelog fragment? For the sanity and tox failures, there is a separate PR, which is yet to be merged. Once that PR is merged, we can rebase and merge this PR. Apart from these minor issues, everything else looks good.

**Comment by GomathiselviS** [comment]
recheck

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**